doc/content/getting_started/installation.md

```diff
@@ -59,20 +59,50 @@ To compile FENIX, first make sure that the conda MOOSE environment is activated
 conda activate moose
 ```
 
-Then navigate to the FENIX clone directory and download the MOOSE submodule:
+Then navigate to the FENIX clone directory and download the MOOSE and TMAP8 submodules:
 
 ```bash
 cd ~/projects/FENIX
 git submodule update --init moose
+git submodule update --init tmap8
 ```
 
-!alert note
-The copy of MOOSE provided with FENIX has been fully tested against the current
+Finally, Cardinal and its dependencies must be downloaded and pre-built. To perform this, first run
+the build script from the FENIX clone directory:
+
+```bash
+cd ~/projects/FENIX
+scripts/update_and_rebuild_cardinal.sh
+```
```

Comment:
```suggestion
```

Cardinal is now built within the Makefile rules rather then being done manually.